lib/matplotlib/table.py

```diff
@@ -86,14 +86,31 @@ def get_fontsize(self):
         'Return the cell fontsize'
         return self._text.get_fontsize()
 
-    def auto_set_font_size(self, renderer):
+    def auto_set_font_size(self, renderer, size_hint=None, grow=False):
         """ Shrink font size until text fits. """
+
+        if size_hint:
```

Comment:
Why do this in here?  It is just as easy to set the initial font just before calling this method.